src/g4vg_impl/Converter.cc

```diff
@@ -308,6 +310,18 @@ auto Converter::build_with_daughters(G4LogicalVolume const* mother_g4lv)
                 place_daughter(g4pv);
             }
         }
+        else if (auto* replica = dynamic_cast<G4PVReplica*>(g4pv))
+        {
+            // TODO: check and error if the replica uses the kRaxis replication
+            G4ReplicaNavigation replica_navigator;
+            for (size_type j = 0, jmax = replica->GetMultiplicity(); j != jmax;
+                 ++j)
+            {
+                replica_navigator.ComputeTransformation(j, replica);
+                replica->SetCopyNo(j);
```

Comment:
Does the copy number for the replicas actually get changed in Geant4? (I'm not seeing that it does.) It seems to me the replicated daughters should all be "the same" and have the same copy number as the `replica` here. The same should apply to the parameterised volume above.

Reply:
I could be _totally_ misunderstanding here as I was going by the navigation code here: 

https://gitlab.cern.ch/geant4/geant4/-/blob/master/source/geometry/navigation/src/G4Navigator.cc?ref_type=heads#L230-235

and as we're replacing an effective single PV with individual (VecGeom) PVs, it seemed important that the copy number is retained. Ultimately the important bit for AdePT/Celeritas is ensuring the steps/touchables sent from device to host are correct so any SD gets correct copy numbers on the G4(host) side. If the copy number isn't important for that, this could be removed.

Reply:
Ah good call, I wouldn't have expected the copy number being set there of all places. I'll open another quick PR to do the same for the "placement" volumes above since it looks like the navigator does that too.